src/appkit.cs

```diff
@@ -11391,7 +11391,8 @@ interface NSPathControlItem
 	}
 
 	[BaseType (typeof (NSResponder))]
-	interface NSPopover : NSAccessibilityElementProtocol, NSAccessibility {
+	interface NSPopover : NSAppearanceCustomization, NSAccessibilityElementProtocol, NSAccessibility {
+		[Obsolete ("Use GetAppearance () and SetAppearance () methods instead")]
```

Comment:
I think this needs custom formatting for the IDEs to display proper errors, @VincentDondain?